readme.md

```diff
@@ -44,7 +44,9 @@ Import the `github-markdown.css` file and add a `markdown-body` class to the con
 </article>
 ```
 
-If you want code syntax highlighted, use GitHub Flavored Markdown rendered from [GitHub's `/markdown` API](https://docs.github.com/en/free-pro-team@latest/rest/reference/markdown).
+You can use [GitHub's `/markdown` API](https://docs.github.com/en/free-pro-team@latest/rest/reference/markdown) to turn markdown into the HTML that GitHub generates, which works well with the CSS generated here. Other markdown parsers will mostly work with these styles too. To mimic how GitHub highlights code, you can use [`starry-night`](https://github.com/wooorm/starry-night) with your markdown parser of choice.
+
+If you want code syntax highlighted, use GitHub Flavored Markdown rendered from .
```

Comment:
"from" where?

Reply:
Oh whoops that's the old line I think that was left from before. I remove it

Reply:
remvoed